### DISCUSSION TITLE
Import error notifications.py

### DISCUSSION THREAD

**Discussion by becausebirds** [mcguirepr89/BirdNET-Pi#736] [Q&A] [answered]
Yesterday my station stopped analyzing and recording, the logs just showed the following message on loop:

<img width="1398" alt="Screenshot 2023-02-09 at 5 55 21 PM" src="https://user-images.githubusercontent.com/110354170/218173613-270f5f0f-5da5-40f0-9255-d4bb0da9d2c4.png">

I was curious about the notifications.py file and inspected the file to find it completely blank. This didn't seem right, so I pulled the latest copy of that file from GitHub and just pasted all the contents back into my copy of the file and things seemed to begin working again.

Any idea what could cause this problem? I think it may have happened while I performed an update.

**Comment by ehpersonal38** [comment, marked as answer]
Agh, updates seem to be breaking things a lot recently.

But this is a new one for me, I don't know why an update would empty a single file... I'd chalk it up to Git having some kind of bug or something
